From 158b6485ef9fbf1ef35ecd54388a7e331c645817 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Mon, 19 Aug 2013 17:53:07 +0800
Subject: [PATCH 239/602] ENGR00273616 OTA build failure in all fsl platform

Use ubi device node for fstab nand.
Recovery will use device node to format the cache/data partition

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 hdmidongle_6dq/fstab_nand.freescale | 8 ++++----
 sabreauto_6q/fstab_nand.freescale   | 8 ++++----
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/hdmidongle_6dq/fstab_nand.freescale b/hdmidongle_6dq/fstab_nand.freescale
index 4e22c5a..b35fee9 100644
--- a/hdmidongle_6dq/fstab_nand.freescale
+++ b/hdmidongle_6dq/fstab_nand.freescale
@@ -7,9 +7,9 @@
 
 /devices/platform/sdhci-esdhc-imx.1/mmc_host/mmc1 /mnt/extsd vfat defaults voldmanaged=sdcard:auto
 /devices/platform/fsl-ehci /mnt/udisk vfat defaults voldmanaged=sdcard:auto
-ubi0:system	/system		ubifs 	ro					wait
-ubi0:data       /data           ubifs   nosuid,nodev,nodiratime,noatime         wait
-ubi0:cache      /cache          ubifs   nosuid,nodev                            wait
-ubi0:device     /device         ubifs   nosuid,nodev                            wait
+/dev/ubi0_0     /system		ubifs 	ro					wait
+/dev/ubi0_3     /data           ubifs   nosuid,nodev,nodiratime,noatime         wait
+/dev/ubi0_1     /cache          ubifs   nosuid,nodev                            wait
+/dev/ubi0_2     /device         ubifs   nosuid,nodev                            wait
 /dev/mtd/mtd1   /boot           mtd     defaults                                defaults
 /dev/mtd/mtd2   /recovery       mtd     defaults                                defaults
diff --git a/sabreauto_6q/fstab_nand.freescale b/sabreauto_6q/fstab_nand.freescale
index 24d20ba..efe94f6 100644
--- a/sabreauto_6q/fstab_nand.freescale
+++ b/sabreauto_6q/fstab_nand.freescale
@@ -7,9 +7,9 @@
 
 /devices/platform/sdhci-esdhc-imx.0/mmc_host/mmc1 /mnt/extsd vfat defaults voldmanaged=sdcard:auto
 /devices/platform/fsl-ehci /mnt/udisk vfat defaults voldmanaged=sdcard:auto
-ubi0:system	    /system		    ubifs 	ro                                      wait
-ubi0:data       /data           ubifs   nosuid,nodev,nodiratime,noatime         wait
-ubi0:cache      /cache          ubifs   nosuid,nodev                            wait
-ubi0:device     /device         ubifs   nosuid,nodev                            wait
+/dev/ubi0_0     /system		    ubifs 	ro                                      wait
+/dev/ubi0_3     /data           ubifs   nosuid,nodev,nodiratime,noatime         wait
+/dev/ubi0_1     /cache          ubifs   nosuid,nodev                            wait
+/dev/ubi0_2     /device         ubifs   nosuid,nodev                            wait
 /dev/mtd/mtd1   /boot           mtd     defaults                                defaults
 /dev/mtd/mtd2   /recovery       mtd     defaults                                defaults
-- 
1.8.0

